dts/arm/nxp/nxp_mcxw71.dtsi

```diff
@@ -57,7 +57,7 @@
 
 			stcm0: system_memory@0 {
 				compatible = "mmio-sram";
-				reg = <0x0 DT_SIZE_K(64)>;
+				reg = <0x0 DT_SIZE_K(104)>;
```

Comment:
maybe at least put a comment saying that the first 64 KB are ecc and the last 40 are not

Reply:
I can make a comment on the next update.

Reply:
you made an update, but didn't make the comment